etc/org.opencastproject.userdirectory.canvas.CanvasUserRoleProvider.cfg.template

```diff
@@ -0,0 +1,29 @@
+# Canvas UserDirectoryProvider configuration
+
+# This is an an optional service which is not enabled by default. To enable it,
+# edit etc/org.apache.karaf.features.cfg and add opencast-canvas to the featuresBoot option.
+
+# The organization for this provider
+# default: mh_default_org
+org.opencastproject.userdirectory.canvas.org=mh_default_org
```

Comment:
It might be a good idea to clarify in the docs that this provider only supports a single organization.  IOW if you are hosting multiple orgs then you can only have one hooked up to Canvas.